test/unit/testPaymentBreakdown.js

```diff
@@ -2,10 +2,25 @@
 
 const initSteps = require('app/core/initSteps');
 const assert = require('chai').assert;
+const config = require('test/config');
 const co = require('co');
 
 describe('PaymentBreakdown', () => {
     const steps = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]);
+    let s2sStub, payStub;
+
+    before(() => {
+        config.s2sStubErrorSequence = '01';
+        s2sStub = require('test/service-stubs/idam');
+        payStub = require('test/service-stubs/payment');
+    });
+
+    after(() => {
+        s2sStub.close();
+        payStub.close();
+        delete require.cache[require.resolve('test/service-stubs/idam')];
+        delete require.cache[require.resolve('test/service-stubs/payment')];
+    });
```

Comment:
We don't need to use the stubs in these unit tests, instead we can mock the service calls to return the data that we expect.